default.nix

```diff
@@ -1,7 +1,7 @@
 { system ? builtins.currentSystem, pkgs ? import ./nix { inherit system; } }:
 pkgs.buildGoApplication rec {
   pname = "cronosd";
-  version = "0.5.2";
+  version = "0.5.4-testnet";
```

Comment:
why the version name has a `testnet` postfix? it doesn't match with **the line 6**

Reply:
its related to the release version